lib/matplotlib/table.py

```diff
@@ -146,6 +147,67 @@ def set_text_props(self, **kwargs):
         self._text.update(kwargs)
 
 
+class CustomCell(Cell):
+    """
+    A subclass of Cell where the sides may be visibly toggled.
+
+    """
+
+    _edges = 'BRTL'
+    _edge_aliases = {'open':         '',
```

Comment:
@tacaswell do you mean the extra whitespace inside the _edge_aliases dictionary? I can't see any other whitespace and yeah pep8 isn't failing.